components/Footer/Components/MissionLogo.jsx

```diff
@@ -12,10 +12,10 @@ export default function MissionLogo({ url, mission }) {
     <div className="su-flex su-flex-col lg:su-flex-row">
       <a
         href={url}
-        className="pre-footer-logo dark:pre-footer-logo-dark su-mr-[44px] su-max-w-[231px] lg:su-max-w-[296px] lg:su-mt-[6px] su-flex-shrink-0"
+        className="pre-footer-logo dark:pre-footer-logo-dark su-mr-44 su-max-w-[231px] lg:su-max-w-[296px] lg:su-mt-6 su-shrink-0"
```

Comment:
su-flex-shrink, su-flex-shrink-0, su-flex-grow, su-flex-grow-0 classes are deprecated - no longer need the flex in the classname:
https://tailwindcss.com/docs/flex-grow